src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -262,7 +262,7 @@ async def Generate(
                 log_tracing_disabled_warning()
             generators.append(
                 self.engine.generate(
-                    inputs=inputs,
+                    inputs,
```

Comment:
I'd rather not remove this, I prefer explicitly specifying the names of the input, as this has caused issues in the past

Reply:
can I move `inputs` to `prompt` and assume `>v0.6.1.post2` or do I have to be backward compatible?

Reply:
Ah, sad this wasn't done in a non-breaking way in vllm: https://github.com/vllm-project/vllm/pull/8673/files#diff-e8f334a55697b72398acbbed4d280bcb3e5eb6c2e66fc81ccde477818b4b0351R38. Maybe we can simply comment here that the kwarg name is removed due to this change, and then in later releases add in `prompt=`?

Might also be worth changing the name of the `inputs` variable here to `prompt`

Reply:
We are reverting upstream for now https://github.com/vllm-project/vllm/pull/8750.

Reply:
I guess it could've been done in a less abrupt way upstream by warning about deprecation of `inputs` while adding both `inputs` and `prompts` for some time..